rtg_tools/build/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM java:8-jdk
+
+MAINTAINER Trevor Pesout, tpesout@ucsc.edu
+
+RUN apt-get update && apt-get install -y \
+    git \
+	ant \
+	unzip
+
+# Create a new source directory
+WORKDIR /home
+
+# Get RTG tools
+WORKDIR /home/rtg-tools
+RUN git clone https://github.com/RealTimeGenomics/rtg-tools.git
+WORKDIR /home/rtg-tools/rtg-tools
+RUN git checkout 8867750ed351b92ac983fa38f98635fd52c98fd5
```

Comment:
Any chance they have a tag of the version you want?  I.e. https://github.com/BD2KGenomics/cgl-docker-lib/pull/259/commits/2ee7e46ecace40f726a483f6cd0e3371158fe883